src/partition/repart.c

```diff
@@ -3924,6 +3965,11 @@ static int partition_format_verity_hash(
                 node = partition_target_path(t);
         }
 
+        if (p->verity_data_block_size == UINT64_MAX)
+                p->verity_data_block_size = context->sector_size;
+        if (p->verity_hash_block_size == UINT64_MAX)
+                p->verity_hash_block_size = context->sector_size;
```

Comment:
You should rebase on latest main and then update these to use `context->fs_sector_size` instead

Reply:
Done.